utils/ilstr_jpeg_lepton_compress/src/ilstr_jpeg_lepton_compress/__init__.py

```
@@ -0,0 +1,169 @@
+sentry_sdk.profiler.start_profiler()
+
+def get_image_id(dbCursor, image_path) -> int|None:
+    path_info = Path(image_path)
+    absolute_path = str(path_info.absolute())
+
+    try:
+        dbCursor.execute("SELECT id FROM illusts WHERE path = %s LIMIT 1", (absolute_path,))
+        if dbCursor.rowcount < 1:
+            return None
+        return dbCursor.fetchone()['id']
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)